features/mbedtls/targets/TARGET_NUVOTON/TARGET_M480/sha/sha_alt_hw.c

```diff
@@ -424,7 +419,7 @@ void crypto_sha_getinternstate(unsigned char output[], size_t olen)
     uint32_t *in_pos = (uint32_t *) &CRPT->HMAC_DGST[0];
     unsigned char *out_pos = output;
     uint32_t rmn = olen;
-    
+
     while (rmn) {
```

Comment:
This is pre-existing, but we must guard against accidental passing of non-aligned `olen` here. Currently, the code goes into an infinite loop and unbounded overflow in this case.

Reply:
@hanno-arm I guard against it in https://github.com/ARMmbed/mbed-os/pull/4925/commits/c853f32f0aac936a6abbf11f86a4d58e45500262.